### PR TITLE
CDI-690 - update assertions and introduce new requestContext tests in

### PR DESCRIPTION
**Pull request by tremes**
SE.

### CODE REVIEW

File: impl/src/main/java/org/jboss/cdi/tck/tests/se/context/request/TestBean.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import javax.enterprise.inject.spi.BeanManager;
+import javax.inject.Inject;
+
+@ApplicationScoped
+public class TestBean {
+
+    @Inject
+    BeanManager bm;
+
+    private AtomicBoolean isReqContextActive = new AtomicBoolean(false);
```
Comment (mkouba) [inline impl/src/main/java/org/jboss/cdi/tck/tests/se/context/request/TestBean.java]: `isRequestContextActiveDuringPostConstruct` might be a better name...

Comment (tremes) [inline impl/src/main/java/org/jboss/cdi/tck/tests/se/context/request/TestBean.java]: Well yes. Thanks

File: impl/src/main/java/org/jboss/cdi/tck/tests/se/context/request/TestBean.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    @PostConstruct
+    public void init(){
+        isReqContextActive.set(bm.getContext(RequestScoped.class).isActive());
```
Comment (mkouba) [inline impl/src/main/java/org/jboss/cdi/tck/tests/se/context/request/TestBean.java]: I would rather catch the `ContextNotActiveException` here and set the value to false. The reason is that the exception might lead to `CreationException` and that might complicate test debugging.

Comment (tremes) [inline impl/src/main/java/org/jboss/cdi/tck/tests/se/context/request/TestBean.java]: OK good point.

**Comment by tremes** [comment]
Merged. Thanks for review.

**Pull request closed without merging**